props/prop_base.h

```diff
@@ -514,9 +514,9 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
     SetPreset(0, true);
   }
 
-#ifdef BLADE_DETECT_PIN
+// #ifdef BLADE_DETECT_PIN
```

Comment:
Why comment out the #ifdef ?

Reply:
I am finishing up the edits to my prop - needed blade_detected_ in there, but doing without it instead.
The aboves above will revert in a moment.

Reply:
done.